src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx

```diff
@@ -306,3 +308,299 @@ function nearestCompletelyFillingOrder(price, marketDepth) {
+      .attr('dx', 0)
+      .attr('dy', drawParams.chartDim.tickOffset)
+      .text('min')
+
+    rangeBounds.append('rect')
+      .attr('class', 'bounding-box')
+      .attr('x', 0)
+      .attr('y', () => drawParams.yScale(orderBookKeys.min))
+      .attr('height', drawParams.drawHeight - drawParams.yScale(orderBookKeys.min))
+      .attr('width', drawParams.containerWidth)
+
+  } else if (drawParams.yDomain[drawParams.yDomain.length - 1] > marketMax) {
+    rangeBounds.append('line')
+      .attr('class', 'tick-line')
+      .attr('x1', 0)
+      .attr('x2', drawParams.containerWidth)
+      .attr('y1', () => drawParams.yScale(orderBookKeys.max))
+      .attr('y2', () => drawParams.yScale(orderBookKeys.max))
+
+    rangeBounds.append('text')
+      .attr('class', 'tick-value')
+      .attr('x', 0)
+      .attr('y', d => drawParams.yScale(orderBookKeys.max))
+      .attr('dx', 0)
+      .attr('dy', drawParams.chartDim.tickOffset)
+      .text('max')
+
+    rangeBounds.append('rect')
+      .attr('class', 'bounding-box')
+      .attr('x', 0)
+      .attr('y', 0)
+      .attr('height', drawParams.yScale(orderBookKeys.max))
+      .attr('width', drawParams.containerWidth)
+  }
+
+  // X Axis
+  depthChart.append('g')
+    .attr('id', 'depth-x-axis')
+    .attr('transform', `translate(0, ${drawParams.containerHeight - drawParams.chartDim.bottom})`)
+    .call(d3.axisBottom(drawParams.xScale)
+      .ticks(3))
+    .select('path').remove()
+}
+
+function drawLines(options) {
+  const {
+    drawParams,
+    depthChart,
+    marketDepth,
+  } = options
+
+  // Defs
+  const chartDefs = depthChart.append('defs')
+
+  //  Fills
+  const subtleGradient = chartDefs.append('linearGradient')
+    .attr('id', 'subtleGradient')
+
+  subtleGradient.append('stop')
+    .attr('class', 'stop-left')
+    .attr('offset', '0')
+
+  subtleGradient.append('stop')
+    .attr('class', 'stop-right')
+    .attr('offset', '1')
+
+  // Depth Line
+  const depthLine = d3.line()
+    .curve(d3.curveStepAfter)
+    .x(d => drawParams.xScale(d[0]))
+    .y(d => drawParams.yScale(d[1]))
+
+  Object.keys(marketDepth).forEach((side) => {
+    depthChart.append('path')
+      .data([marketDepth[side]])
+      .attr('class', `depth-line outcome-line-${side}`)
+      .attr('d', depthLine)
+  })
+
+  const area = d3.area()
+    .curve(d3.curveStepAfter)
+    .x0(0)
+    .x1(d => drawParams.xScale(d[0]))
+    .y(d => drawParams.yScale(d[1]))
+
+  Object.keys(marketDepth).forEach((side) => {
+    depthChart.append('path')
+      .data([marketDepth[side]])
+      .classed('filled-subtle', true)
+      .attr('d', area)
+  })
+}
+
+function setupCrosshairs(options) {
+  const {
+    depthChart,
+  } = options
+
+  depthChart.append('text')
+    .attr('id', 'hovered_price_label')
+
+  // create crosshairs
+  const crosshair = depthChart.append('g')
+    .attr('id', 'crosshairs')
+    .attr('class', 'line')
+    .attr('style', { display: 'none' })
```

Comment:
Can/Should this be `.style()`?

Reply:
Ah, yep, all utilizations should be migrated to `style` — will do that!